cpp/src/gandiva/precompiled/arithmetic_ops.cc

```diff
@@ -234,6 +234,37 @@ DIV_FLOAT(float64)
 
 #undef DIV_FLOAT
 
+#define BITWISE_AND(TYPE)                                                  \
```

Comment:
you could use existing BINARY_SYMMETRIC macro for AND, OR

Reply:
Done.